core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -170,16 +169,11 @@ long memorySize() {
 
         engaged.forEach(Chunk::release);
 
-        if (result.success && result.putIfAbsent) {
-            size.incrementAndGet();
-        }
-
-        return result;
     }
 
     private void checkRebalance(Chunk c) {
```

Comment:
This function is almost never called. (only remove)
When should we check if a rebalance is needed? with new put implementation rebalance is done only when out of space. is this ok? @Eshcar @eranmeir @sanastas